cmd/iam.go

```diff
@@ -1136,7 +1136,7 @@ func (sys *IAMSys) NewServiceAccount(ctx context.Context, parentUser string, gro
 	}
 	cred.ParentUser = parentUser
 	cred.Groups = groups
-	cred.Status = string(madmin.AccountEnabled)
+	cred.Status = string(auth.AccountOn)
```

Comment:
This is a breaking change btw @vadmeste  - how will the IsValid work here?

Reply:
> This is a breaking change btw @vadmeste - how will the IsValid work here?

This is not a breaking change, the service account PR which was merged yesterday was wrong. It was always 'on' to indicate that a credentials is enabled.

Reply:
okay cool @vadmeste